vs-build/FASTlib/FASTlib.vfproj

```diff
@@ -117,43 +117,76 @@
 		<Filter Name="Resource Files" Filter="rc;ico;cur;bmp;dlg;rc2;rct;bin;rgs;gif;jpg;jpeg;jpe">
 		<File RelativePath="..\RunRegistry.bat">
 			<FileConfiguration Name="Debug_Double|Win32" ExcludedFromBuild="true"/>
-			<FileConfiguration Name="Debug|Win32" ExcludedFromBuild="true"/>
 			<FileConfiguration Name="Release_Double|Win32" ExcludedFromBuild="true"/>
+			<FileConfiguration Name="Debug|Win32" ExcludedFromBuild="true"/>
```

Comment:
For the sake of fewer merge conflict headaches in the future, could you remove these unnecessary changes related to generating the types files? (i.e., revert the changes where Visual Studio decides to change the order of the `FileConfiguration` entries even though nothing has been changed on them? I really wish Visual Studio would stop doing that.) That way the only changes in this file are the additional files you added + the compiler settings you modified.

Reply:
Certainly.  It's really annoying that VS keeps doing that.

Reply:
Thanks. 

Also, I noticed that you added the OpenMP directive on all the configurations in these projects. When I was helping with FAST.Farm, we made a separate configuration for OpenMP because--while it builds--it doesn't run properly with OpenMP + static libraries. So, we left the original configurations for static builds and made the OpenMP version use dynamic libraries. I don't see that you did that here, though. Did you test the executables that this generates? Wondering if this has changed.

Reply:
@ebranlard can you address the OpenMP VS issue?

Reply:
Done!  Thanks @ebranlard